src/main/java/de/retest/recheck/ignore/CompoundFilter.java

```diff
@@ -37,4 +47,8 @@ public boolean matches( final Element element, final AttributeDifference attribu
 		return filters;
 	}
 
+	public void add( final Filter filter ) {
```

Comment:
I think we should strive for immutability here too. If `RecheckOptions` allows something like:

```java
RecheckOptions opts = RecheckOptions.builder()
        .filter( f0, f1, f2)
        .build();
```

I think there is no need for this.